tsplib95/fields.py

```diff
@@ -238,7 +238,7 @@ class AdjacencyListField(TransformerField):
     def build_transformer(cls):
         return T.MapT(key=T.FuncT(func=int),
                       value=T.ListT(value=T.FuncT(func=int)),
-                      sep="\n",
+                      sep=('-1', ' -1\n'),
```

Comment:
I'm not sure what the implication is, but the conflict is on this line. I have
```
value=T.ListT(value=T.FuncT(func=int), terminal='-1'),
sep='\n',
```
and you have
```
value=T.ListT(value=T.FuncT(func=int)),
sep=('-1', ' -1\n'),
```

Subtle difference but I don't know what the implication is...